roles/etcd/meta/main.yml

```diff
@@ -18,3 +18,4 @@ galaxy_info:
 dependencies:
 - { role: os_firewall }
 - { role: etcd_common }
+- { role: etcd_server_certificates }
```

Comment:
I'm not opposed to joining them back together but I liked the idea of them being separate with this code since they didn't really share any code. If we consolidate CA into a single role then I think it makes sense to combine them again.